### PR TITLE
Made video on scholorship page responsive

### PR DESCRIPTION
**Pull request by sethbergman**
#596 
Fixed merge issues @rickr

### CODE REVIEW

File: app/views/pages/scholarships.html.erb [hunk 1]
```diff
@@ -5,8 +5,8 @@
         <p class="lead">At Operation Code, we partner with both code schools and industry leaders to make transitioning to software development less painful and accessible for our nation's finest and their families.</p>
       </div>
 
-      <div class="center">
-        <iframe src="https://player.vimeo.com/video/125189545" width="700" height="481" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+      <div class="video-responsive">
```
Comment (jjhampton) [inline app/views/pages/scholarships.html.erb]: I thought we were aiming to use the same class for all the videos on the site - either  `.video-responsive` or `.video-container` for all of them?

Comment (sethbergman) [inline app/views/pages/scholarships.html.erb]: OMG
I did use the old class.

**Comment by sethbergman** [comment]
Should be good to go now. I've been all over the place today. My wife was in a wreck last night, she's ok though. :pray:

**Comment by sethbergman** [comment]
I'll handle this tomorrow. Not sure what happened with the merge conflict.

**Pull request closed without merging**